settings.yml

```
@@ -3,3 +3,8 @@ enabled_categories:
   list_type: category
   default: ""
   description: "Choose which categories that users need to sign up for."
+enabled_tags:
+  type: list
+  list_type: tag
+  default: "gated"
```

Let's make the default empty, so it definitely doesn't affect existing sites

```suggestion
  default: ""
```